components/atoms/VerticalStepper.js

```diff
@@ -6,7 +6,7 @@ import fr from '../../locales/fr'
 const VerticalLinearStepper = (props) => {
   const t = props.locale === 'en' ? en : fr
   const statusColor =
-    props.benefitStatus === 'Active' ? 'bg-green-active' : 'bg-orange-pending'
+    props.benefitStatus === 'Active' ? 'bg-green-active' : 'bg-gray-pending'
```

Comment:
I believe this file is deleted in dashboard. Pull latest from dashboard and file should be gone

Reply:
Done!